generate_docker.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+"""Generated Dockerfile from a template
+"""
+
+import argparse
+import configuration
+
+# The template file name for Dockerfile
+DOCKERFILE_TEMPLATE_FILE_NAMES = ["Dockerfile.template"]
+
+# The default docker image to use
+DEFAULT_DOCKER_IMAGE = 'agpipeline/gantry-base-image:latest'
```

Comment:
I would suggest not using latest, but an actual version number here. This template is made for a specific version of the base image.

Reply:
I always go back and forth on this. My thinking is that since we're using TravisCI for testing (coming soon) we should be OK with the latest as a general rule. If something breaks, then the number either needs to be pinned down (no active development), or a fix made (actively being developed ).  If something is to be put on "hold" for any reason, then a simple checkin would lock the version. I think what's missing from the Dockerfile.template is a version number for the template.